backwards-compatability/node10/package.json

```diff
@@ -12,7 +12,7 @@
     "@opentelemetry/tracing": "0.23.0"
   },
   "devDependencies": {
-    "@types/node": "10.17.60",
```

Comment:
backwards-compat configs should stay on the corresponding major version

Reply:
Well sorry i was tired i guess i though it didn't update those. Can't we just close the PR since it only want to update them ?

Reply:
@dyladan tries since a while to convince renovate-bot to do what we want. Not sure if he wants to try again.

Reply:
I'm getting pretty tired of trying, but it will just open another PR next time a major version is released (which admittedly doesn't happen that often)

Reply:
Actually @types/node 15.x and 16.x exist meanwhile. But it seems bot puts all it's work into updating backward-compat to v14.